src/parser/types.ts

```diff
@@ -63,6 +63,7 @@ export interface SurveyComponent extends StudyComponent {
 }
 
 export interface Trial {
+  meta?: object;
```

Comment:
Can you change this to Record<string, any> type? 

Using object as type leads to issues. A record expects the key will be string, but object requires named keys, so it only allows using inbuilt keys.